spec/models/orchestration_stack_spec.rb

```diff
@@ -0,0 +1,7 @@
+require "spec_helper"
+
+RSpec.describe OrchestrationStack do
+  it "can runs a compliance test" do
+    expect(described_class.instance_methods).to include(:check_compliance_queue)
```

Comment:
Could this be better expressed with `respond_to?`? Even so, I don't see a lot of value in these kinds of tests. Could we add one that tests the behavior instead?